src/Spatial/Euclidean/Plane.cs

```diff
@@ -74,6 +77,30 @@ public Plane(Point3D rootPoint, UnitVector3D normal)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Plane"/> struct.
+        /// Constructs a Plane from the given points.
+        /// </summary>
+        /// <param name="points">The points</param>
+        /// <returns>Fitted <see cref="Plane"/></returns>
+        /// <remarks>this method uses SVD to fit the plane.</remarks>
+        public static Plane CreateFittedPlaneFrom(IEnumerable<Point3D> points)
+        {
+            var c = Point3D.Centroid(points);
+
+            var A = CreateMatrix.DenseOfRowVectors(
+                points.Select(p => p - c)
+                    .Select(p => CreateVector.DenseOfArray(new double[] { p.X, p.Y, p.Z })));
```

Comment:
What I would do in this case, is to first allocate the necessary matrix space (`new DenseMatrix(rows, columns)`), and then write a single for-each to populate each row of it. This way we could easily parallelize the loop in case we need it, if for example we have a LOT of points.

Reply:
will do, thanks.
I think that the parallelization is out of scope for me.
Thanks again for your point of view.

Reply:
Does this fix follow the policy of Math.NET Spatial?
I'm afraid the case where you maintainers respect RAII([Resource acquisition is initialization](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization)) principle and I would break it.

Reply:
> I think that the parallelization is out of scope for me.

Sure, I wasn't expecting this to be part of this PR. I don't want to prematurely optimize things in any case.

Reply:
> Does this fix follow the policy of Math.NET Spatial?
> I'm afraid the case where you maintainers respect RAII([Resource acquisition is initialization](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization)) policy and I would break it.

I am not sure I understand your question/concern here, can you explain it again please?